projects/lua/luaL_addgsub_test-afl++.toml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Sergey Bronnikov
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "-l debug -j 2"
+target = "/luaL_addgsub_test_sydr @@"
```

Comment:
Remove `_test` from all names of binaries, except for libfuzzer binaries in all .toml files

Reply:
Done.

```diff
diff --git a/projects/lua/luaL_addgsub_test-afl++.toml b/projects/lua/luaL_addgsub_test-afl++.toml
index 840b8dc..e687998 100644
--- a/projects/lua/luaL_addgsub_test-afl++.toml
+++ b/projects/lua/luaL_addgsub_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_addgsub_test_sydr @@"
+target = "/luaL_addgsub_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_addgsub"
-target = "/luaL_addgsub_test_afl @@"
+target = "/luaL_addgsub_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_addgsub_test_cmplog -t 60000 -i /corpus_luaL_addgsub"
-target = "/luaL_addgsub_test_afl @@"
+args = "-m none -l AT -c /luaL_addgsub_cmplog -t 60000 -i /corpus_luaL_addgsub"
+target = "/luaL_addgsub_afl @@"
 
 [cov]
-target = "/luaL_addgsub_test_cov @@"
+target = "/luaL_addgsub_cov @@"
diff --git a/projects/lua/luaL_addgsub_test.toml b/projects/lua/luaL_addgsub_test.toml
index 4a4dc33..7566299 100644
--- a/projects/lua/luaL_addgsub_test.toml
+++ b/projects/lua/luaL_addgsub_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_addgsub_test_sydr @@"
+target = "/luaL_addgsub_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_addgsub_test"
 args = "/corpus_luaL_addgsub"
 
 [cov]
-target = "/luaL_addgsub_test_cov @@"
+target = "/luaL_addgsub_cov @@"
diff --git a/projects/lua/luaL_buffaddr_test-afl++.toml b/projects/lua/luaL_buffaddr_test-afl++.toml
index ad74fec..ab11a95 100644
--- a/projects/lua/luaL_buffaddr_test-afl++.toml
+++ b/projects/lua/luaL_buffaddr_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_buffaddr_test_sydr @@"
+target = "/luaL_buffaddr_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_buffaddr"
-target = "/luaL_buffaddr_test_afl @@"
+target = "/luaL_buffaddr_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_buffaddr_test_cmplog -t 60000 -i /corpus_luaL_buffaddr"
-target = "/luaL_buffaddr_test_afl @@"
+args = "-m none -l AT -c /luaL_buffaddr_cmplog -t 60000 -i /corpus_luaL_buffaddr"
+target = "/luaL_buffaddr_afl @@"
 
 [cov]
-target = "/luaL_buffaddr_test_cov @@"
+target = "/luaL_buffaddr_cov @@"
diff --git a/projects/lua/luaL_buffaddr_test.toml b/projects/lua/luaL_buffaddr_test.toml
index ce96c8c..eb728e8 100644
--- a/projects/lua/luaL_buffaddr_test.toml
+++ b/projects/lua/luaL_buffaddr_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_buffaddr_test_sydr @@"
+target = "/luaL_buffaddr_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_buffaddr_test"
 args = "/corpus_luaL_buffaddr"
 
 [cov]
-target = "/luaL_buffaddr_test_cov @@"
+target = "/luaL_buffaddr_cov @@"
diff --git a/projects/lua/luaL_bufflen_test-afl++.toml b/projects/lua/luaL_bufflen_test-afl++.toml
index 89538eb..78fc26b 100644
--- a/projects/lua/luaL_bufflen_test-afl++.toml
+++ b/projects/lua/luaL_bufflen_test-afl++.toml
@@ -16,12 +16,12 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_bufflen_test_sydr @@"
+target = "/luaL_bufflen_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_bufflen"
-target = "/luaL_bufflen_test_afl @@"
+target = "/luaL_bufflen_afl @@"
 
 [[aflplusplus]]
 args = "-m none -l AT -c /luaL_bufflen_test_cmplog -t 60000 -i /corpus_luaL_bufflen"
diff --git a/projects/lua/luaL_bufflen_test.toml b/projects/lua/luaL_bufflen_test.toml
index 2a0315a..257da4c 100644
--- a/projects/lua/luaL_bufflen_test.toml
+++ b/projects/lua/luaL_bufflen_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_bufflen_test_sydr @@"
+target = "/luaL_bufflen_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_bufflen_test"
 args = "/corpus_luaL_bufflen"
 
 [cov]
-target = "/luaL_bufflen_test_cov @@"
+target = "/luaL_bufflen_cov @@"
diff --git a/projects/lua/luaL_buffsub_test-afl++.toml b/projects/lua/luaL_buffsub_test-afl++.toml
index a220289..18b51ff 100644
--- a/projects/lua/luaL_buffsub_test-afl++.toml
+++ b/projects/lua/luaL_buffsub_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_buffsub_test_sydr @@"
+target = "/luaL_buffsub_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_buffsub"
-target = "/luaL_buffsub_test_afl @@"
+target = "/luaL_buffsub_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_buffsub_test_cmplog -t 60000 -i /corpus_luaL_buffsub"
-target = "/luaL_buffsub_test_afl @@"
+args = "-m none -l AT -c /luaL_buffsub_cmplog -t 60000 -i /corpus_luaL_buffsub"
+target = "/luaL_buffsub_afl @@"
 
 [cov]
-target = "/luaL_buffsub_test_cov @@"
+target = "/luaL_buffsub_cov @@"
diff --git a/projects/lua/luaL_buffsub_test.toml b/projects/lua/luaL_buffsub_test.toml
index 1d7a6f7..b91de51 100644
--- a/projects/lua/luaL_buffsub_test.toml
+++ b/projects/lua/luaL_buffsub_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_buffsub_test_sydr @@"
+target = "/luaL_buffsub_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_buffsub_test"
 args = "/corpus_luaL_buffsub"
 
 [cov]
-target = "/luaL_buffsub_test_cov @@"
+target = "/luaL_buffsub_cov @@"
diff --git a/projects/lua/luaL_dostring_test-afl++.toml b/projects/lua/luaL_dostring_test-afl++.toml
index 09ce03b..66166ee 100644
--- a/projects/lua/luaL_dostring_test-afl++.toml
+++ b/projects/lua/luaL_dostring_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_dostring_test_sydr @@"
+target = "/luaL_dostring_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_dostring"
-target = "/luaL_dostring_test_afl @@"
+target = "/luaL_dostring_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_dostring_test_cmplog -t 60000 -i /corpus_luaL_dostring"
-target = "/luaL_dostring_test_afl @@"
+args = "-m none -l AT -c /luaL_dostring_cmplog -t 60000 -i /corpus_luaL_dostring"
+target = "/luaL_dostring_afl @@"
 
 [cov]
-target = "/luaL_dostring_test_cov @@"
+target = "/luaL_dostring_cov @@"
diff --git a/projects/lua/luaL_dostring_test.toml b/projects/lua/luaL_dostring_test.toml
index 630f557..19f8b9e 100644
--- a/projects/lua/luaL_dostring_test.toml
+++ b/projects/lua/luaL_dostring_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_dostring_test_sydr @@"
+target = "/luaL_dostring_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_dostring_test"
 args = "/corpus_luaL_dostring"
 
 [cov]
-target = "/luaL_dostring_test_cov @@"
+target = "/luaL_dostring_cov @@"
diff --git a/projects/lua/luaL_gsub_test-afl++.toml b/projects/lua/luaL_gsub_test-afl++.toml
index 14cc2f1..a68ef3e 100644
--- a/projects/lua/luaL_gsub_test-afl++.toml
+++ b/projects/lua/luaL_gsub_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_gsub_test_sydr @@"
+target = "/luaL_gsub_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_gsub"
-target = "/luaL_gsub_test_afl @@"
+target = "/luaL_gsub_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_gsub_test_cmplog -t 60000 -i /corpus_luaL_gsub"
-target = "/luaL_gsub_test_afl @@"
+args = "-m none -l AT -c /luaL_gsub_cmplog -t 60000 -i /corpus_luaL_gsub"
+target = "/luaL_gsub_afl @@"
 
 [cov]
-target = "/luaL_gsub_test_cov @@"
+target = "/luaL_gsub_cov @@"
diff --git a/projects/lua/luaL_gsub_test.toml b/projects/lua/luaL_gsub_test.toml
index d11e9c1..2a7979a 100644
--- a/projects/lua/luaL_gsub_test.toml
+++ b/projects/lua/luaL_gsub_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_gsub_test_sydr @@"
+target = "/luaL_gsub_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_gsub_test"
 args = "/corpus_luaL_gsub"
 
 [cov]
-target = "/luaL_gsub_test_cov @@"
+target = "/luaL_gsub_cov @@"
diff --git a/projects/lua/luaL_loadbuffer_proto_test.toml b/projects/lua/luaL_loadbuffer_proto_test.toml
index 218742f..7f3c873 100644
--- a/projects/lua/luaL_loadbuffer_proto_test.toml
+++ b/projects/lua/luaL_loadbuffer_proto_test.toml
@@ -20,7 +20,7 @@ target = "/luaL_loadbuffer_sydr @@"
 jobs = 2
 
 [libfuzzer]
-path = "/luaL_loadbuffer_fuzzer"
+path = "/luaL_loadbuffer_test"
 args = "/corpus_luaL_loadbuffer"
 
 [cov]
diff --git a/projects/lua/luaL_loadbuffer_test-afl++.toml b/projects/lua/luaL_loadbuffer_test-afl++.toml
index 157b0d8..2b6780c 100644
--- a/projects/lua/luaL_loadbuffer_test-afl++.toml
+++ b/projects/lua/luaL_loadbuffer_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadbuffer_test_sydr @@"
+target = "/luaL_loadbuffer_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_loadbuffer"
-target = "/luaL_loadbuffer_test_afl @@"
+target = "/luaL_loadbuffer_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_loadbuffer_test_cmplog -t 60000 -i /corpus_luaL_loadbuffer"
-target = "/luaL_loadbuffer_test_afl @@"
+args = "-m none -l AT -c /luaL_loadbuffer_cmplog -t 60000 -i /corpus_luaL_loadbuffer"
+target = "/luaL_loadbuffer_afl @@"
 
 [cov]
-target = "/luaL_loadbuffer_test_cov @@"
+target = "/luaL_loadbuffer_cov @@"
diff --git a/projects/lua/luaL_loadbuffer_test.toml b/projects/lua/luaL_loadbuffer_test.toml
index 69c7d9b..2c7cbef 100644
--- a/projects/lua/luaL_loadbuffer_test.toml
+++ b/projects/lua/luaL_loadbuffer_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadbuffer_test_sydr @@"
+target = "/luaL_loadbuffer_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_loadbuffer_test"
 args = "/corpus_luaL_loadbuffer"
 
 [cov]
-target = "/luaL_loadbuffer_test_cov @@"
+target = "/luaL_loadbuffer_cov @@"
diff --git a/projects/lua/luaL_loadbufferx_test-afl++.toml b/projects/lua/luaL_loadbufferx_test-afl++.toml
index 8bab203..bc0edeb 100644
--- a/projects/lua/luaL_loadbufferx_test-afl++.toml
+++ b/projects/lua/luaL_loadbufferx_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadbufferx_test_sydr @@"
+target = "/luaL_loadbufferx_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_loadbufferx"
-target = "/luaL_loadbufferx_test_afl @@"
+target = "/luaL_loadbufferx_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_loadbufferx_test_cmplog -t 60000 -i /corpus_luaL_loadbufferx"
-target = "/luaL_loadbufferx_test_afl @@"
+args = "-m none -l AT -c /luaL_loadbufferx_cmplog -t 60000 -i /corpus_luaL_loadbufferx"
+target = "/luaL_loadbufferx_afl @@"
 
 [cov]
-target = "/luaL_loadbufferx_test_cov @@"
+target = "/luaL_loadbufferx_cov @@"
diff --git a/projects/lua/luaL_loadbufferx_test.toml b/projects/lua/luaL_loadbufferx_test.toml
index f5b3ff3..791a365 100644
--- a/projects/lua/luaL_loadbufferx_test.toml
+++ b/projects/lua/luaL_loadbufferx_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadbufferx_test_sydr @@"
+target = "/luaL_loadbufferx_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_loadbufferx_test"
 args = "/corpus_luaL_loadbufferx"
 
 [cov]
-target = "/luaL_loadbufferx_test_cov @@"
+target = "/luaL_loadbufferx_cov @@"
diff --git a/projects/lua/luaL_loadstring_test-afl++.toml b/projects/lua/luaL_loadstring_test-afl++.toml
index 5623311..60f76e1 100644
--- a/projects/lua/luaL_loadstring_test-afl++.toml
+++ b/projects/lua/luaL_loadstring_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadstring_test_sydr @@"
+target = "/luaL_loadstring_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_loadstring"
-target = "/luaL_loadstring_test_afl @@"
+target = "/luaL_loadstring_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_loadstring_test_cmplog -t 60000 -i /corpus_luaL_loadstring"
-target = "/luaL_loadstring_test_afl @@"
+args = "-m none -l AT -c /luaL_loadstring_cmplog -t 60000 -i /corpus_luaL_loadstring"
+target = "/luaL_loadstring_afl @@"
 
 [cov]
-target = "/luaL_loadstring_test_cov @@"
+target = "/luaL_loadstring_cov @@"
diff --git a/projects/lua/luaL_loadstring_test.toml b/projects/lua/luaL_loadstring_test.toml
index a3f7dba..d4dd762 100644
--- a/projects/lua/luaL_loadstring_test.toml
+++ b/projects/lua/luaL_loadstring_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_loadstring_test_sydr @@"
+target = "/luaL_loadstring_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_loadstring_test"
 args = "/corpus_luaL_loadstring"
 
 [cov]
-target = "/luaL_loadstring_test_cov @@"
+target = "/luaL_loadstring_cov @@"
diff --git a/projects/lua/luaL_traceback_test-afl++.toml b/projects/lua/luaL_traceback_test-afl++.toml
index 0232ae8..0fe6ae3 100644
--- a/projects/lua/luaL_traceback_test-afl++.toml
+++ b/projects/lua/luaL_traceback_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_traceback_test_sydr @@"
+target = "/luaL_traceback_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_luaL_traceback"
-target = "/luaL_traceback_test_afl @@"
+target = "/luaL_traceback_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /luaL_traceback_test_cmplog -t 60000 -i /corpus_luaL_traceback"
-target = "/luaL_traceback_test_afl @@"
+args = "-m none -l AT -c /luaL_traceback_cmplog -t 60000 -i /corpus_luaL_traceback"
+target = "/luaL_traceback_afl @@"
 
 [cov]
-target = "/luaL_traceback_test_cov @@"
+target = "/luaL_traceback_cov @@"
diff --git a/projects/lua/luaL_traceback_test.toml b/projects/lua/luaL_traceback_test.toml
index 0199caf..502fb3d 100644
--- a/projects/lua/luaL_traceback_test.toml
+++ b/projects/lua/luaL_traceback_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/luaL_traceback_test_sydr @@"
+target = "/luaL_traceback_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/luaL_traceback_test"
 args = "/corpus_luaL_traceback"
 
 [cov]
-target = "/luaL_traceback_test_cov @@"
+target = "/luaL_traceback_cov @@"
diff --git a/projects/lua/lua_load_test-afl++.toml b/projects/lua/lua_load_test-afl++.toml
index dd7a846..b1499df 100644
--- a/projects/lua/lua_load_test-afl++.toml
+++ b/projects/lua/lua_load_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/lua_load_test_sydr @@"
+target = "/lua_load_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_lua_load"
-target = "/lua_load_test_afl @@"
+target = "/lua_load_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /lua_load_test_cmplog -t 60000 -i /corpus_lua_load"
-target = "/lua_load_test_afl @@"
+args = "-m none -l AT -c /lua_load_cmplog -t 60000 -i /corpus_lua_load"
+target = "/lua_load_afl @@"
 
 [cov]
-target = "/lua_load_test_cov @@"
+target = "/lua_load_cov @@"
diff --git a/projects/lua/lua_load_test.toml b/projects/lua/lua_load_test.toml
index 493c86f..2314e5e 100644
--- a/projects/lua/lua_load_test.toml
+++ b/projects/lua/lua_load_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/lua_load_test_sydr @@"
+target = "/lua_load_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/lua_load_test"
 args = "/corpus_lua_load"
 
 [cov]
-target = "/lua_load_test_cov @@"
+target = "/lua_load_cov @@"
diff --git a/projects/lua/lua_stringtonumber_test-afl++.toml b/projects/lua/lua_stringtonumber_test-afl++.toml
index 0ed262a..4c94e51 100644
--- a/projects/lua/lua_stringtonumber_test-afl++.toml
+++ b/projects/lua/lua_stringtonumber_test-afl++.toml
@@ -16,16 +16,16 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/lua_stringtonumber_test_sydr @@"
+target = "/lua_stringtonumber_sydr @@"
 jobs = 2
 
 [[aflplusplus]]
 args = "-t 60000 -i /corpus_lua_stringtonumber"
-target = "/lua_stringtonumber_test_afl @@"
+target = "/lua_stringtonumber_afl @@"
 
 [[aflplusplus]]
-args = "-m none -l AT -c /lua_stringtonumber_test_cmplog -t 60000 -i /corpus_lua_stringtonumber"
-target = "/lua_stringtonumber_test_afl @@"
+args = "-m none -l AT -c /lua_stringtonumber_cmplog -t 60000 -i /corpus_lua_stringtonumber"
+target = "/lua_stringtonumber_afl @@"
 
 [cov]
-target = "/lua_stringtonumber_test_cov @@"
+target = "/lua_stringtonumber_cov @@"
diff --git a/projects/lua/lua_stringtonumber_test.toml b/projects/lua/lua_stringtonumber_test.toml
index 62eec1e..2f9efb2 100644
--- a/projects/lua/lua_stringtonumber_test.toml
+++ b/projects/lua/lua_stringtonumber_test.toml
@@ -16,7 +16,7 @@
 
 [sydr]
 args = "-l debug -j 2"
-target = "/lua_stringtonumber_test_sydr @@"
+target = "/lua_stringtonumber_sydr @@"
 jobs = 2
 
 [libfuzzer]
@@ -24,4 +24,4 @@ path = "/lua_stringtonumber_test"
 args = "/corpus_lua_stringtonumber"
 
 [cov]
-target = "/lua_stringtonumber_test_cov @@"
+target = "/lua_stringtonumber_cov @@"
```